relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -59,6 +59,39 @@ fn extract_http_method(transaction: &Event) -> Option<String> {
     Some(method.clone())
 }
 
+/// Extract the browser name from the [`Context::Browser`] context.
+fn extract_browser_name(event: &Event) -> Option<String> {
+    let contexts = event.contexts.value()?;
+    let browser = contexts.get("browser").and_then(Annotated::value);
+    if let Some(ContextInner(Context::Browser(browser_context))) = browser {
+        return browser_context.name.value().cloned();
+    }
```

Comment:
Out of curiosity, are these already sanitized/normalized, or can this yield 17 ways of saying "Chrome"?
Same question for OS.